src/pairwiseFunctors/Functor.h

```diff
@@ -145,4 +147,48 @@ class Functor {
   virtual bool allowsNonNewton3() { return false; }
 };
 
-}  // namespace autopas
\ No newline at end of file
+/**
+ * Macro to define the SoALoaders.
+ * @param cell name for cell like parameter
+ * @param soa name for soa
+ * @param offset name for offset
+ * @param body the actual function body for the soa loader
+ *
+ * @note generates two loaders, one for verlet lists, one for the normal case.
+ * @note the need for this could be removed if the soa's are removed from the particlecells (highly unlikely)
+ */
+#define AUTOPAS_FUNCTOR_SOALOADER(cell, soa, offset, body)                                                        \
+  void SoALoader(ParticleCell &cell, SoA<SoAArraysType> &soa, size_t offset = 0) override { body }                \
+  /** @copydoc SoALoader(ParticleCell &cell, SoA<SoAArraysType> &soa, size_t offset) */                           \
+  template <typename /*dummy*/ = void,                                                                            \
```

Comment:
what is the first template argument for?

Reply:
as it's said, it's a dummy :)
removed it, still works. -> removed